### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by benarent**
Updates Readme to better reflect rebrand https://goteleport.com/blog/gravitational-is-teleport/ and Application Access

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
-# Gravitational Teleport
+# Teleport
 
-Gravitational Teleport is a modern security gateway for remotely accessing:
+Teleport is a modern security gateway for remotely accessing:
 * Clusters of Linux servers via SSH or SSH-over-HTTPS in a browser.
```
Comment (klizhentas) [inline README.md:4]: Let's make all those links actually:

* [Clusters of Linux Servers](...) via SSH ...
* [Kuberntes Clusters](...)
* [Web applications](...)
* [Databases](preview link(

File: README.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
-# Gravitational Teleport
+# Teleport
 
-Gravitational Teleport is a modern security gateway for remotely accessing:
+Teleport is a modern security gateway for remotely accessing:
```
Comment (klizhentas) [inline README.md:3]: Teleport is a unified access plane for infrastructure:

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (benarent) [reply README.md]: Updated these links, once we've updated Teleport Server section I'll update the URL for the first 'cluster of linux servers' for now I' sent this to our quickstart

File: README.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+* [Databases - (Postgres Preview)](https://goteleport.com/teleport/docs/preview/teleport-database-access/)
 
 It is intended to be used instead or together with `sshd` for organizations who
 need:
 
 * SSH audit with session recording/replay.
 * Kubernetes API Access with audit and `kubectl exec` recording/replay.
 * Easily manage trust between teams, organizations and data centers.
-* Have SSH or Kubernetes access to behind-firewall clusters without any open ports.
+* Have Application, SSH or Kubernetes access to behind-firewall clusters without any open ports.
```
Comment (webvictim) [inline README.md:16]: The "have" here is redundant in the context of the header i.e. "organisations who need <x>".

```suggestion
* Application, SSH or Kubernetes access to behind-firewall clusters without any open ports.
```

File: README.md [hunk 4]
```diff
@@ -19,9 +22,9 @@ because it facilitates easy adoption of the best infrastructure security
 practices like:
 
 - No need to distribute keys: Teleport uses certificate-based access with automatic certificate expiration time.
-- 2nd factor authentication (2FA) for SSH and Kubernetes.
+- 2nd factor authentication (2FA) for Apps, SSH, Kubernetes.
 - Collaboratively troubleshoot issues through session sharing.
-- Single sign-on (SSO) for SSH/Kubernetes and your organization identities via
+- Single sign-on (SSO) for Apps,SSH/Kubernetes and your organization identities via
```
Comment (webvictim) [inline README.md:27]:
```suggestion
- Single sign-on (SSO) for Applications, SSH/Kubernetes and your organization identities via
```

File: README.md [hunk 5]
```diff
@@ -144,37 +147,37 @@ some common tasks:
 
 Latest version:
 
-```
+```sh
```
Comment (webvictim) [inline README.md:150]: Why `sh` rather than `bash`?
```suggestion
```bash
```

File: README.md [hunk 6]
```diff
@@ -19,9 +22,9 @@ because it facilitates easy adoption of the best infrastructure security
 practices like:
 
 - No need to distribute keys: Teleport uses certificate-based access with automatic certificate expiration time.
-- 2nd factor authentication (2FA) for SSH and Kubernetes.
+- 2nd factor authentication (2FA) for Apps, SSH, Kubernetes.
```
Comment (webvictim) [inline README.md:25]:
```suggestion
- 2nd factor authentication (2FA) for Apps, SSH and Kubernetes.
```

File: README.md [hunk 7]
```diff
@@ -31,8 +34,8 @@ implementation and it is _fully compatible with OpenSSH_ and can be used with
 
 |Project Links| Description
 |---|----
-| [Teleport Website](https://gravitational.com/teleport)  | The official website of the project |
-| [Documentation](https://gravitational.com/teleport/docs/quickstart/)  | Admin guide, user manual and more |
+| [Teleport Website](https://goteleport.com/teleport)  | The official website of the project |
```
Comment (webvictim) [inline README.md:37]:
```suggestion
| [Teleport Website](https://goteleport.com/teleport) | The official website of the project |
```

File: README.md [hunk 8]
```diff
@@ -31,8 +34,8 @@ implementation and it is _fully compatible with OpenSSH_ and can be used with
...
+| [Documentation](https://goteleport.com/teleport/docs/quickstart/)  | Admin guide, user manual and more |
```
Comment (webvictim) [inline README.md:38]:
```suggestion
| [Documentation](https://goteleport.com/teleport/docs/quickstart/) | Admin guide, user manual and more |
```

**Review by webvictim — APPROVED** [review]

**Review by webvictim — APPROVED** [review]

**Pull request merged**